src/map/utils/mobutils.cpp

```diff
@@ -928,8 +928,10 @@ void SetupNMMob(CMobEntity* PMob)
 
 void SetupMaat(CMobEntity* PMob)
 {
-    // Should swing normally when not a mnk
-    PMob->m_Weapons[SLOT_MAIN]->setDelay((240*1000)/60);
+    // I arrived at this attack speed value by comparing his attacks to
+    // videos of retail Maat.  I believe it's base attack rate of 480
+    // minus six levels of martial arts (-20 each)
```

Comment:
According to the original comment this is for non mnk Maat so I'm betting mnk Maat uses the 300 in the db.

From my comment in the open issue about maat:
> Here's the actual line screwing with Maats melee delay. 
`PMob->m_Weapons[SLOT_MAIN]->setDelay((240*1000)/60);`
Prior to that in the DB he's set to exactly 300.
```INSERT INTO `mob_pools` VALUES (2460,'Maat',335,0x0000FA0200000000000000000000000000000000,2,2,1,300,100,0,0,0,0,18,0,0,0,3,0,0,0,1,0,335);```
The majority of monk mobs are set to 360. A few are different values. No idea if they should all have the same delay or not. If there are mistakes now, we can bet on people copy and pasting those mistakes to new mobs later. 
>
> If there are "tiers" for mob h2h delay there should probably be some guidance added to the wiki about it.  (perhaps it was because of MA traits on mnk mobs? Does that even apply with mobs? If so I'd wager that being the reason for any having wrong melee speed when its not the zero delay problem) 

So I suggest editing that sql line as well.